### ISSUE TITLE
Rate-limit comments/issues opens

### ISSUE THREAD

**Issue by techknowlogick** [go-gitea/gitea#9785]
For public gitea instances this would prevent users coming in and creating many issues in a short period of time.

cc: @ashimokawa @Codeberg-org

**Comment by zeripath** [comment]
Whilst we don't have this option in it might be worth people sticking rate limits on posts to the appropriate URLs.

**Comment by lunny** [comment]
Don't you mean APIs but UIs?

**Comment by Codeberg-org** [comment]
@lunny : yes. Attacks were executed by manual creation of single-use-accounts (entering captcha and activating account by email via disposable email provider), then issues+comments created via API, IP addresses changing every few requests (so no avail for fail2ban).

Since UI is scriptable easily using for example selenium, the UI has the same potential issues (albeit it was not used this time).

**Comment by wxiaoguang** [comment]
It seems that there is no more interest in this topic.
